src/chrome/content/rules/Peer1.ca.xml

```diff
@@ -1,19 +0,0 @@
-<!--
-	Nonfunctional subdomains:
-
-		- ^	(refused)
-		- www	(redirects to http, valid cert)
-
--->
-<ruleset name="Peer1.ca (partial)">
-
-	<target host="*.peer1.com" />
-
-
-	<securecookie host="^(?:www\.)?partners\.peer1\.ca$" name=".+" />
-
-
-	<rule from="^http://(www\.)?partners\.peer1\.ca/"
-		to="https://$1partners.peer1.ca/" />
```

Comment:
note https://partners.peer1.com (.com vs .ca) does exist, however it redirects to https://www.cogecopeer1.com/en/partners/ which redirects to https://aptum.com/en/partners/ which is 404

But this rule never worked, so there is little reason to try rescuing it by creating a Peer1.com  and/or cogecopeer1.com ruleset for previous operating names